docker-setup.sh

```diff
@@ -6,12 +6,13 @@ set -e
 
 pull_docker_snapshot() {
   project="${1?project name required}"
+  stack_version="$2"
```

Comment:
Just to make problem-solving easier in the future, as long as we're rebinding to a usefully named var, we should blow up helpfully if the now-required positioned var is missing.

```suggestion
  stack_version="${2?stack version required}"
```